### PR TITLE
libbtbb: add devel subport tracking upstream master

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update, claim maintainership (it is my port initially).

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.8.5
Xcode 5.1.1

macOS 10.6.8
Xcode 3.2.6

macOS 10A190
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by barracuda156** [comment]
Wait a bit, I need to add Cmake config installation.

File: devel/libbtbb/Portfile [hunk 1]
```diff
@@ -4,21 +4,26 @@ PortSystem          1.0
...
 description         Bluetooth baseband decoding library.
 long_description    Bluetooth baseband decoding library, forked from the GR-Bluetooth project. \
                     It can be used to extract Bluetooth packet and piconet information \
                     from Ubertooth devices as well as GR-Bluetooth/USRP.
 license             GPL-2
-maintainers         nomaintainer
-categories          devel net
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
 
 homepage            https://github.com/greatscottgadgets/libbtbb
```
Comment (reneeotten) [inline devel/libbtbb/Portfile:18]: this is set by the github PG

**Review by reneeotten — CHANGES_REQUESTED** [review]
one minor suggestion

File: devel/libbtbb/Portfile (hunk 1 above)
Comment (barracuda156) [reply devel/libbtbb/Portfile]: Fixed!

**Comment by barracuda156** [comment]
@reneeotten This also should be mergeable.

**Comment by barracuda156** [comment]
@reneeotten Could we merge this, so that `ubertooth` build can be confirmed?

**Pull request merged**